alf/metrics/metrics.py

```diff
@@ -325,21 +325,27 @@ def __init__(self,
                  dtype=torch.float32,
                  discount=0.99,
                  reward_transformer=None,
-                 buffer_size=10):
+                 buffer_size=10,
+                 reward_clip=None):
```

Comment:
`reward_transformer=RewardClipping()` can achieve the same effect.

Reply:
Removed reward_clip.